src/io/DxfData.cc

```diff
@@ -419,7 +419,7 @@ DxfData::DxfData(double fn, double fs, double fa,
         auto lv = grid.data(this->points[lines[idx].idx[j]][0], this->points[lines[idx].idx[j]][1]);
         for (size_t ki = 0; ki < lv.size(); ++ki) {
           int k = lv.at(ki);
-          if (k < 0 || k >= lines.size()) {
+          if (k < 0 || k >= static_cast<int>(lines.size())) {
```

Comment:
This inequality can be cast the other undefined-behavior-safer way, k to unsigned int, since k is guarded as not less than 0.